packages/rdf/lib/localFetch/localFetch.js

```diff
@@ -75,13 +76,17 @@ async function localFetch(
     https: fetchHTTPWithMeta,
   })
 
-  const url = isAbsolute(input)
-    ? input
-    : basePath
-      ? `file://${resolve(basePath, input)}`
-      : input
+  try {
+    return fetch(new URL(input).toString())
+  } catch (e) {
+    const url = isAbsolute(input)
+      ? input
+      : basePath
+        ? pathToFileURL(resolve(basePath, input)).toString()
+        : pathToFileURL(input).toString()
 
-  return fetch(url)
+    return fetch(url)
+  }
```

Comment:
I may suggest adding a comment to explain that part, so that we can know why it was done that way in the future.